### ISSUE TITLE
[Playground - HTML]:Elements that have scrollable content must be accessible by keyboard.[Sev3][WCAG2.1.1][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#15]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to 'Playground' link and activate it.
4) Now select 'Some advanced features' from 'Choose a sample' combo box and navigate to 'HTML' Tab and activate it.
5) Run the Accessibility Insights for web tool and observe the issue.

**Actual Result:**
Elements that have scrollable content is not accessible by keyboard.

**Expected Result:**
Elements that have scrollable content should be accessible by keyboard.
Control(s) must be accessible to keyboards and other assistive technology. Common causes of this problem include a) the element does not have a proper role assigned, b) the element needs tab-index="0" attribute to be focusable, or c) the component is not registering keypresses. 

**User Impact:**
When interactive components are not accessible, assistive technology users are blocked, which leads to everything from inaccessible content, features, and functionality, up to entire applications or sites.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard
<img width="960" alt="Elements that have scrollable content must be accessible by keyboard" src="https://user-images.githubusercontent.com/85154417/121009131-6efac880-c7b1-11eb-9d28-0be7ae0395b1.png">
